usecase/usecase-wrt-android-tests/samples/SharedModeLibraryDownload/index.html

```diff
@@ -0,0 +1,83 @@
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Wang, Hongjuan <hongjuanx.wang@intel.com>
+
+-->
+
+<meta charset="utf-8" />
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width">
+<link rel="stylesheet" type="text/css" href="../../css/bootstrap.css">
+<link rel="stylesheet" type="text/css" href="../../css/main.css">
+<script src="../../js/jquery-2.1.3.min.js"></script>
+<script src="../../js/bootstrap.min.js"></script>
+<script src="../../js/common.js"></script>
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+  <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <h4>Precondition</h4>
+  <ol>
+    <li>Setup a web server and make sure the test device can connect it</li>
+    <li>Make sure the test device has not install XWalkRuntimeLib.apk</li>
+  </ol>
+  <h4>Test Steps</h4>
+  <ol>
+    <li>Build a webapp with command:<br/>
+        $ python make_apk.py --package=[package] --name=[name] --app-url=https://crosswalk-project.org/ --xwalk-apk-url=[http://host/XWalkRuntimeLib.apk] --mode=shared --project-dir=[dirName]
+    </li>
+    <li>Open the AndroidManifest.xml from directory [dirname] and check if there has define a meta-data element with the name "xwalk_apk_url" inside the application tag</li>
+    <li>Install and launch it</li>
+    <li>Click "Get Crosswalk", check the screen</li>
+    <li>Click "Install", then check the screen</li>
+    <li>Click "Done", then check the screen</li>
+    <li>Change the value of "xwalk-apk-url" in res/manifest.json to your server address, then build a webapp with below command:<br/>
```

Comment:
Code haven't check the point of "[From AndroidManifest.xml]", we should have 3 ways to test this feature. "Define a meta-data element with the name "xwalk_apk_url" inside the application tag", you can create a app without this feature, and update  AndroidManifest.xml to add it:
<application android:name="org.xwalk.core.XWalkApplication">
    <meta-data android:name="xwalk_apk_url"
               android:value="http://host/XWalkRuntimeLib.apk" />
    ...
</application>